build.gradle

```diff
@@ -60,6 +60,8 @@ allprojects {
 
     repositories {
         mavenCentral()
+        mavenLocal()
```

Comment:
Is this change intended?

Reply:
I suggested this for testing, but once we merge Discord4J/discord-json#43 it won't be needed.